modules/apps/document-library/document-library-web/src/main/java/com/liferay/document/library/web/internal/display/context/logic/UIItemsBuilder.java

```diff
@@ -1400,6 +1525,20 @@ public boolean isPublishActionAvailable() throws PortalException {
 		return true;
 	}
 
+	public boolean isRevertToVersionActionAvailable() throws PortalException {
+		if ((_fileVersion.getStatus() != WorkflowConstants.STATUS_APPROVED) ||
+			!_fileEntryDisplayContextHelper.hasUpdatePermission()) {
+
+			return false;
+		}
+
+		FileVersion latestFileVersion = _fileEntry.getLatestFileVersion();
+
+		String latestFileVersionVersion = latestFileVersion.getVersion();
+
+		return !latestFileVersionVersion.equals(_fileVersion.getVersion());
```

Comment:
In general, boolean expressions need to be used as part of a conditional statement a return an explicit true/false value. You'll need to change this to:
```
if (Objects.equals(latestFileVersion.getVersion(), _fileVersion.getVersion()) {
  return false;
}
else {
  return true;
}
```

Reply:
Ok, I will change that